training/general/src/environments/acpt-pytorch-1.11-py38-cuda11.3-gpu/environment.yaml

```diff
@@ -0,0 +1,32 @@
+image:
+  name: azureml/curated/acpt-pytorch-1.11-py38-cuda11.3-gpu
```

Comment:
Before we proceed with this environment we'll need to turn off the [pre-registry version](https://msdata.visualstudio.com/Vienna/_git/AzureMlCli?path=/envs/Latest/AzureML-PTA-pytorch-1.11-py38-cuda11.3-gpu/.contact).

Reply:
checked in the change for turning off the pre-registry version